entity-framework/core/change-tracking/identity-resolution.md

```diff
@@ -0,0 +1,665 @@
+---
+title: Identity Resolution - EF Core
+description: Resolving multiple entity instances into a single instance using primary key values
+author: ajcvickers
+ms.date: 12/30/2020
+uid: core/change-tracking/identity-resolution
+---
+
+# Identity Resolution in EF Core
+
+A <xref:Microsoft.EntityFrameworkCore.DbContext> can only track one entity instance with any given primary key value. This means multiple instances of an entity with the same key value must be resolved to a single instance. This is called "identity resolution". Identity resolution ensures Entity Framework Core (EF Core) is tracking a consistent graph with no ambiguities about the relationships or property values of the entities.
+
+> [!TIP]
+> This document assumes that entity states and the basics of EF Core change tracking are understood. See [Change Tracking in EF Core](xref:core/change-tracking/index) for more information on these topics.
+
+> [!TIP]  
+> You can run and debug into all the code in this document by [downloading the sample code from GitHub](https://github.com/dotnet/EntityFramework.Docs/tree/master/samples/core/ChangeTracking/IdentityResolutionInEFCore).
+
+## Introduction
+
+The following code queries for an entity and then attempts to attach a different instance with the same primary key value:
+
+<!--
+            using var context = new BlogsContext();
+
+            var blogA = context.Blogs.Single(e => e.Id == 1);
+            var blogB = new Blog { Id = 1, Name = ".NET Blog (All new!)" };
+
+            try
+            {
+                context.Update(blogB); // This will throw
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
+            }
+-->
+[!code-csharp[Identity_Resolution_in_EF_Core_1](../../../samples/core/ChangeTracking/IdentityResolutionInEFCore/IdentityResolutionSamples.cs?name=Identity_Resolution_in_EF_Core_1)]
+
+Running this code results in the following exception:
+
+> System.InvalidOperationException: The instance of entity type 'Blog' cannot be tracked because another instance with the key value '{Id: 1}' is already being tracked. When attaching existing entities, ensure that only one entity instance with a given key value is attached.
```

Comment:
Should this be monospaced, i.e. surrounded with backticks?

Reply:
I think it reads better like this, but happy to change if the team agrees.
/cc @dotnet/efteam